yalafi/packages/cleveref.py

```diff
@@ -0,0 +1,153 @@
+
+#
+#   YaLafi module for LaTeX package cleveref
+#
+
+from yalafi.defs import InitModule, Macro, VoidToken
+from yalafi import utils
+import re
+import sys
+
+
+require_packages = []
+
+re_ref = re.compile(r's/\\(\\cref|\\Cref)\{([^\}\{]+)\}/(.*)/g')
+re_ref_range = re.compile(r's/\\(\\crefrange|\\Crefrange)\{([^\}\{]+)\}\{([^\}\{]+)\}/(.*)/g')
+re_command = re.compile(r'''
+s/\\
+(                   # group 1: command
+    \\
+    (               # group 2: capturing cC
+        \[cC\]
+    )?
+    [a-zA-Z@]+
+)
+(                   # group 3: possible arguments
+    (?:\{\.\*\})+
+)?
+\s*                 # possible whitespaces inbetween
+/
+    (.*)            # group 4: replacement
+/g
+''', re.VERBOSE)
+re_cC = re.compile(r'(\[cC\])')
+re_dot = re.compile(r'\\\.')
+re_double_backslash = re.compile(r'\\\\')
+
+macro_read_sed = '\\LTReadSed'

```

Comment:
Any suggestions for a command that better fits together with the other `\LT…` commands?